### DISCUSSION TITLE
i found a new exploit to (basically) remove extensions

### DISCUSSION THREAD

**Discussion by ILikeBypassing** [3kh0/ext-remover#232] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
you need touchscreen for this to work

1. go to [extensions](chrome://extensions/)
2. click details on the extension you want
3. enable dev mode
4. make a bookmark for the site you want
5. put the chrome extension page and a newtab page on split screen
6. spam "allow file access to urls"
7. click the bookmarked site
8. once on your site spam this bookmarklet: javascript:(function(){window.onbeforeunload=function(){return '';}})();

heres a demonstration
[Screen recording 2022-11-12 10.28.50 AM.webm](https://user-images.githubusercontent.com/118064735/201481423-fdb7dc2b-a72c-426e-b432-2a408e1dd844.webm)
btw i press cancel not leave 

idk if anyone already posted this but im sorry if they did

**Comment by bartholemuew** [comment]
Not a lot of people have dev mode on the chromebook.

**Reply by xHydra1337** [reply]
I think he means dev mode on the extensions page, not the whole chromebook in dev mode.

**Reply by ILikeBypassing** [reply]
watch the video

**Reply by bartholemuew** [reply]
> I think he means dev mode on the extensions page, not the whole chromebook in dev mode.

thats what i mean

**Comment by ghost** [comment]
This is a good alternative to task manager, thanks for pointing this out. You could also see the extension's id by using ltbeef's extension recognition (even though it doesn't work)

**Comment by DodoSeal** [comment]
Hmm I can enable the file access without dev mode. Sadly I can't enable dev mode to make my own extensions.

**Comment by Aka-but-nice** [comment]
Why would people have dev mode available like your school is the definition of "retarded" does your school allow you to use a personal account, I wouldn't be surprised!

**Reply by scatcat5isdead** [reply]
mine does

**Reply by Aka-but-nice** [reply]
dang your school is retarded
